qoqo-macros/src/devices.rs

```diff
@@ -0,0 +1,540 @@
+// Copyright © 2021-2023 HQS Quantum Simulations GmbH. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the
+// License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! qoqo-macros
+//!
+//! Attribute proc-macros for the traits of qoqo [qoqo].
```

Comment:
Just a little thing, but this could be more specific.

Reply:
Ah good catch this shouldn't be here